keps/sig-storage/1491-csi-migration-vsphere/README.md

```diff
@@ -35,6 +40,28 @@ used as as described in its parent KEP. For all other contents, please refer to
   on kubelet and treat vSphere CSI migration as already complete. And directly redirect traffic to CSI 
   driver for all vsphere volume related operations.
 
+## Design Details
+[X] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+### Prerequisite testing updates
+No additional prerequisite tests are needed.
+
+
+### Unit tests
+Unit tests for vSphere CSI Migration is added in the csi-translation-lib repository
+ - https://github.com/kubernetes/csi-translation-lib/blob/master/plugins/vsphere_volume_test.go (85.1% coverage)
+
+### Integration tests
+ - No Integration tests required for this feature
+
+### e2e tests
+ - Existing e2e tests developed for in-tree vSphere volume plugin executed - https://github.com/kubernetes/kubernetes/tree/master/test/e2e/storage/vsphere
```

Comment:
For CSI migration we also want to ensure that tests inside - https://github.com/kubernetes/kubernetes/tree/master/test/e2e/storage/testsuites are passing before and after migration, using intree PVs. Can you please update this section? It is pretty much a requirement for all CSI migration.

Reply:
@gnufied addressed your review comment in this PR - https://github.com/kubernetes/enhancements/pull/3398